src/elements/content-explorer/ContentExplorer.scss

```
@@ -7,5 +7,22 @@
         .bcpr {
             z-index: 1; // Prevents overlay issues with list-item when a file is previewed
         }
+
+        .bce-container {
+            display: flex;
+            height: 100%;
+            min-height: 0;
+        }
```

I don't think you need this `div`, try:
```suggestion
        .be-app-element{
            flex-direction: row;
        }
```
and then you should be able to remove the surrounding div in the JSX